server/game/cards/02_SHD/units/HuntingNexu.ts

```diff
@@ -0,0 +1,23 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Aspect, KeywordName, Location } from '../../../core/Constants';
+
+export default class HuntingNexu extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '8991513192',
+            internalName: 'hunting-nexu'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addConstantAbility({
+            title: 'While you control another Aggression unit, this unit gains Raid 2',
+            condition: (context) => context.source.controller.getOtherUnitsInPlayWithAspect(context.source, Aspect.Aggression).length > 0,
```

Comment:
Do you mind adding a helper similar to the one mentioned above called `Player.isAspectInPlay()`? Probably will be useful for other cases like this in the future